app/_locales/es_419/messages.json

```diff
@@ -528,184 +789,307 @@
+    "message": "Escriba su contraseña para confirmar que es usted."
   },
   "mustSelectOne": {
-    "message": "Se debe seleccionar al menos 1 token."
+    "message": "Debe seleccionar al menos 1 token."
   },
   "myAccounts": {
     "message": "Mis cuentas"
   },
   "myWalletAccounts": {
-    "message": "Las cuentas de mi billetera"
+    "message": "Mis cuentas de cartera"
   },
   "myWalletAccountsDescription": {
-    "message": "Todas tus cuentas creadas en MetaMask se agregarán automáticamente a esta sección."
+    "message": "Todas sus cuentas de MetaMask creadas se agregarán automáticamente a esta sección."
   },
   "needEtherInWallet": {
-    "message": "Necesitarás tener Ethers en tu billetera para poder interactuar con aplicaciones descentralizadas a través de MetaMask."
+    "message": "Para interactuar con aplicaciones descentralizadas mediante MetaMask, necesitará ethers en su cartera."
   },
   "needImportFile": {
-    "message": "Selecciona un archivo para importar.",
-    "description": "User is important an account and needs to add a file to continue"
+    "message": "Debe seleccionar un archivo para la importación."
   },
   "negativeETH": {
-    "message": "No se pueden enviar montos negativos de ETH."
+    "message": "No se pueden enviar cantidades negativas de ETH."
   },
   "networkName": {
     "message": "Nombre de la red"
   },
+  "networkSettingsChainIdDescription": {
+    "message": "El id. de la cadena se usa para firmar transacciones. Debe coincidir con el id. de la cadena que devuelve la red. Puede escribir un número decimal o un número hexadecimal con el prefijo \"0x\", pero el número se mostrará en decimal."
+  },
   "networkSettingsDescription": {
     "message": "Agregar y editar redes RPC personalizadas"
   },
   "networks": {
     "message": "Redes"
   },
   "nevermind": {
-    "message": "No importa"
+    "message": "No es importante"
   },
   "newAccount": {
-    "message": "Nueva cuenta"
+    "message": "Cuenta nueva"
   },
   "newAccountDetectedDialogMessage": {
-    "message": "¡Se detectó una nueva dirección! Haz clic aquí para agregarla a tu libreta de direcciones."
+    "message": "Se detectó una dirección nueva. Haga clic aquí para agregarla a la libreta de direcciones."
   },
   "newAccountNumberName": {
-    "message": "Cuenta $1",
-    "description": "Default name of next account to be created on create account screen"
+    "message": "Cuenta $1"
   },
   "newContact": {
-    "message": "Nuevo contacto"
+    "message": "Contacto nuevo"
   },
   "newContract": {
-    "message": "Nuevo contrato"
+    "message": "Contrato nuevo"
   },
   "newNetwork": {
-    "message": "Nueva red"
+    "message": "Red nueva"
   },
   "newPassword": {
-    "message": "Nueva contraseña (mín. 8 caracteres)"
+    "message": "Contraseña nueva (mín. de 8 caracteres)"
   },
   "newToMetaMask": {
-    "message": "¿Nuevo en MetaMask?"
+    "message": "¿Es nuevo en MetaMask?"
```

Comment:
We are gendering the user as male here. Suggest an alternative phrase such as "¿Recién llegó a MetaMask?" ('Recently arrived at MetaMask?')